cmake/BuildTargets.cmake

```diff
@@ -309,14 +309,9 @@ function (configure_library_targets)
 
   file(GLOB children RELATIVE ${library_root} ${library_root}/*)
 
-  if (APPLE)
-    # Temporarily disable oef builds
-    list(FILTER
-         children
-         EXCLUDE
-         REGEX
-         ".*oef.*")
-  endif (APPLE)
+  # if (APPLE) # Temporarily disable oef builds list(FILTER children EXCLUDE REGEX
```

Comment:
Just remove this if we are adding it back in?

Reply:
I should probably not have committed this.